src/quote/quote.jsx

```diff
@@ -1,64 +1,62 @@
 import React, { Component } from 'react';
+import {fetchQuotes} from '../api';
 import './styles.css'
-import $ from 'jquery';
-export default class BlockQuote extends React.Component {
+class BlockQuote extends Component {
     constructor(props) {
         super(props);
 
         this.state = {
             quotes: [],
-            quote: {},
+            quote: {text:'Java is to JavaScript what Car is to Carpet.', 
+                    author:'A JS Developer'},
+            quoteBoxStyles: {opacity:1}
         }
         this.randomQuote = this.randomQuote.bind(this)
-        this.fetchQuotes = this.fetchQuotes.bind(this)
+        this.setInitialState = this.setInitialState.bind(this)
+        this.fadeIn = this.fadeIn.bind(this)
+        this.fadeOut = this.fadeOut.bind(this)
     }
-    fetchQuotes() {
-        fetch('https://type.fit/api/quotes')
-            .then(data => data.json())
-            .then(data => this.setState({ quotes: data }))
-            .then(this.randomQuote)
+    setQuote(quote){
```

Comment:
- return response from api call
- Make a function of `getQuotes` which will be making the api call and setting the initial state
- call this method in `componentDidMount`


```
  compnentDidMount() {
   onGetQuotes();
  }
  onGetQuotes = () => {
  // make api call
  // get response
// set response in state
}
```